modules/sbt-plugin/src/sbt-test/debug-session/hot-code-reload/build.sbt

```
@@ -0,0 +1,37 @@
+import ch.epfl.scala.debugadapter.sbtplugin.DebugAdapterPlugin
+import ch.epfl.scala.debugadapter.testfmk._
+
+import com.microsoft.java.debug.core.protocol.Events.OutputEvent.Category
+import com.microsoft.java.debug.core.protocol.Types.StackFrame
+
+import java.nio.file.{Files, Path}
+import _root_.io.reactivex.subjects.PublishSubject
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
+
+val scalaV = "3.3.0"
+
+val checkBreakpoint =
+  inputKey[Unit]("Check the breakpoint of a debug session")
+val hotCodeReplace =
+  inputKey[Unit]("Check the hot code reloading of a debug session")
+def source = Def.task((Compile / sources).value.find(_.getName == "A.scala").get.toPath)
+
+def placeBreakpoint = Def.inputTask {
+  val uri = (Compile / startMainClassDebugSession).evaluated
+  implicit val context: TestingContext = TestingContext(source.value, scalaV)
+  DebugState.state = DebugTest.checkWithoutShutdown(uri)(Outputed("A"), Breakpoint(6))
+}
+
+def checkHotCodeReplace = Def.inputTask {
+  DebugTest.runChecks(DebugState.state)(Seq(RedefineClasses(), Outputed("C")))
+}
+
+lazy val hotCodeReload =
+  project
+    .in(file("."))
+    .settings(
+      scalaVersion := scalaV,
+      checkBreakpoint := placeBreakpoint.evaluated,
+      hotCodeReplace := checkHotCodeReplace.evaluated
```

Those names are not really consistent.

```suggestion
      checkBreakpoint := checkBreakpointTask.evaluated,
      checkHotCodeReplace := checkHotCodeReplaceTask.evaluated
```